pages/diary.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Diary</title>
+    <link rel="stylesheet" href="../styles/common.css">
+    <link rel="stylesheet" href="../styles/diary.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100..900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="navbar">
+        <div class="home-button">
+            <a href="../index.html"><button><img src="../images/dashboard_inactive.png" alt="dashboard" height="50px"></button></a></li>
+        </div>
+        <div class="menu-items">
+            <a href="./diary.html"><button class="page-button">Diary</button></a></li>
+            <a href="./meals.html"><button>Meals</button></a></li>
+            <a href="./recipes.html"><button>Recipes</button></a></li>
+            <a href="./stats.html"><button>Stats</button></a></li>
+        </div>
+    </div>
+
+    <div class="wrapper">
+        <div class="content">
+
+        
+        <div class="page-header">
+            <h1>Diary Page</h1>
+        </div>
+
+
+        <div class="diary-section">
+            <div class="diary-heading">
+                <div class="left-arrow">←</div>
+                <div class="diary-day">Today</div>
+                <div class="right-arrow">→</div>
+            </div>
+
+            <div class="diary-meal-container breakfast">
+
+                <div class="diary-entry-recipe">
+                    <div class="diary-content">
+                        <div class="recipe-container">
```

Comment:
Similar blocks from previous file. The good news, in react you will be able to create reusable component